### DISCUSSION TITLE
Small problem in the documentation

### DISCUSSION THREAD

**Discussion by D063520** [eclipse-rdf4j/rdf4j#3816] [General]
Hi,

here:

https://rdf4j.org/documentation/programming/rdfstar/

this line of code:

Model model = Rio.parse(new FileInputStream("/path/to/file.ttls"), "", RDFFORMAT.TURTLESTAR);

should be 

Model model = Rio.parse(new FileInputStream("/path/to/file.ttls"), "", RDFFormat.TURTLESTAR);

NOTE casing! or?

Salut
D063520

**Comment by D063520** [comment]
I found another small problem:

Rio.write(model, new FileOuputStream("/path/to/file.ttls"), "", RDFFORMAT.TURTLESTAR);

it should be FileOutputStream ....

**Comment by barthanssens** [comment]
Hi,

thank you for reporting these issues, I'll create a ticket to fix it

Best regards

Bart
